app/src/test/java/io/apicurio/registry/AbstractRegistryTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.registry;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.stream.Collectors;
+
+/**
+ * Abstract base class for all registry tests.
+ * @author eric.wittmann@gmail.com
+ */
+public abstract class AbstractRegistryTest {
```

Comment:
I guess it's OK if it's named (suffix-ed) Test since it's abstract?
(I remember JUnit having issues with this long ago)

Reply:
Or perhaps not JUnit, perhaps Surefire ... not sure :-)
Hence I usually named such classes as TestBase.